build-tools/scripts/TestApks.targets

```diff
@@ -160,19 +166,49 @@
     />
   </Target>
 
+  <Target Name="DeployTestAabs"
+      Condition=" '@(TestAab)' != '' ">
+    <PropertyGroup>
+      <_KeyStore>..\src\Xamarin.Android.Build.Tasks\Tests\Xamarin.ProjectTools\Resources\Base\test.keystore</_KeyStore>
+      <_KeyAlias>mykey</_KeyAlias>
+      <_KeyPass>android</_KeyPass>
+      <_StorePass>android</_StorePass>
+    </PropertyGroup>
+    <Xamarin.Android.Tools.BootstrapTasks.BundleTool
+        Arguments="build-apks --connected-device --overwrite --mode default --bundle &quot;%(TestAab.Identity)&quot; --output &quot;%(TestAab.Identity).apks&quot;"
+        ContinueOnError="ErrorAndContinue"
+        ToolPath="$(JavaSdkDirectory)\bin"
+        JarPath="$(BundleToolJarPath)"
+        AdbToolExe="$(AdbToolExe)"
+        AdbToolPath="$(AdbToolPath)"
+        KeyStore="$([System.IO.Path]::GetFullPath ('$(_KeyStore)'))"
+        KeyAlias="$(_KeyAlias)"
+        KeyPass="$(_KeyPass)"
+        StorePass="$(_StorePass)"
+    />
+    <Xamarin.Android.Tools.BootstrapTasks.BundleTool
+        Arguments="install-apks --modules _ALL_ --apks &quot;%(TestAab.Identity).apks&quot;"
```

Comment:
We should probably mention what `--modules _ALL_` means somewhere.

Reply:
I put the help text that describes it where it is used, and in `app-bundles.md`.